README.md

```diff
@@ -78,13 +100,31 @@ aws_access_key_id=<AWS_ACCESS_KEY_ID>
 aws_secret_access_key=<AWS_SECRET_ACCESS_KEY>
 ```
 
+Edit `~/.aws/config`
+
+```
+[default]
+region = eu-west-2
+output = json
+
+[profile readonly]
+role_arn = arn:aws:iam::530003481352:role/readonly
+source_profile = default
+mfa_serial = arn:aws:iam::530003481352:mfa/YOURUSERNAME
+
+[profile secreteditor]
+role_arn = arn:aws:iam::530003481352:role/secreteditor
+source_profile = default
+mfa_serial = arn:aws:iam::530003481352:mfa/YOURUSERNAME
+```
+
```

Comment:
Shall we show how to use the profile?
- `aws -profile xxx <command>`
- `AWS_PROFILE=xxx <aws command>` (used for run_in_env)